script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -1788,7 +1795,7 @@ My house is actually a small bakery, and you used to come shop there, remember?
 [sha:dbc9d39b3f08b97ebd66df6ea7572ee2ec54d727]{
 -- Page 111, Offset 41711.
 -- ……ああ。そういう事も、ありえたのかもしれない。
-Yeah... I guess that's something else that could have been.
+Yeah... I suppose that could have happened, someplace, somewhere.
```

Comment:
That's the same thing!!
Even if the latter repetition of this is changed (I think I just felt like being cute anyway), I think this is more evocative.

Reply:
It doesn't come across the same way to me - current suggestion is more like it could have happened (implying it maybe actually _did_) , whereas the framing of another world makes it more clear that it couldn't have.

Reply:
The follow up doesn't mesh with your suggestion, but I can sort of incorporate it like this, but I really don't see the point.
>Yeah... I suppose that could have happened, someplace, somewhere, in another world.

Reply:
No, I was saying to drop the someplace/somewhere - 
> Yeah... I suppose there was a world in which that could have happened.

Reply:
Yes, I know. But I don't want that.

Reply:
I mean, I definitely don't think we should have both, and I also don't think someplace/somewhere is the best phrase here for the reasons listed above. It's also just repetitive? Somewhere, somehow is probably the more common pair, but I still think it isn't the right fit for what he's saying.